packages/web/src/components/pages/TestSuites/TestSuiteDetails/TestSuiteSettings/SettingsVariables/SettingsVariables.tsx

```diff
@@ -1,17 +1,19 @@
 import {FC} from 'react';
 
-import {SystemAccess, useSystemAccess} from '@hooks/useSystemAccess';
-
 import {Variables} from '@organisms/EntityDetails';
 
 import {useUpdateTestSuiteMutation} from '@services/testSuites';
 
-const SettingsVariables: FC = () => (
-  <Variables
-    description="Define environment variables which will be shared across your tests. Variables defined at a Test Suite level will override those defined at a Test level."
-    readOnly={!useSystemAccess(SystemAccess.agent)}
-    useUpdateEntity={useUpdateTestSuiteMutation}
-  />
-);
+import {useEntityDetailsPick} from '@store/entityDetails';
 
+const SettingsVariables: FC = () => {
+  const {details} = useEntityDetailsPick('details');
```

Comment:
As in comment somewhere above, we could consider using the `useSystemAccess` to, to avoid moment when the cluster is down, but the UI doesn't have information that the resource is read-only